action/protocol/staking/read_state.go

```diff
@@ -119,6 +121,15 @@ func readStateCandidateByAddress(ctx context.Context, cc CandidateCenter,
 	return c.toIoTeXTypes(), nil
 }
 
+func readStateTotalStakingAmount(ctx context.Context, bp *BucketPool,
+	_ *iotexapi.ReadStakingDataRequest_TotalStakingAmount) (*iotextypes.AccountMeta, error) {
+	meta := iotextypes.AccountMeta{}
+	meta.Address = address.StakingBucketPoolAddr
+	meta.Balance = bp.Total().String()
+	meta.NumActions = bp.Count()
```

Comment:
maybe don't use numactions this way. we should avoid reuse field which are not the same

Reply:
fixed